http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -60,130 +60,146 @@ display_time() {
 }
 
 #####################################################################################################
```

Comment:
I pulled some refactoring of `gce-init.sh` from https://github.com/DataBiosphere/leonardo/pull/1680. Unfortunately it makes the diff pretty large, but the only real change is the new `STRATUM_*` variables.